mne/utils/numpy.py

```diff
@@ -0,0 +1,34 @@
+# Authors: The MNE-Python contributors.
+# License: BSD-3-Clause
+# Copyright the MNE-Python contributors.
+
+import io
+import os
+
+import numpy
+
+AnyFile = str | bytes | os.PathLike | io.IOBase
+
+
+def numpy_fromfile(
```

Comment:
similar comment here as with the file name. `numpy_fromfile` is too similar to `numpy.fromfile` and may lead to devs misreading the source code. Maybe call the func something more descriptive like `read_from_file_or_buffer`?  Open to other (shorter) suggestions.